src/docs/product/integrations/integration-platform/ui-components/alert-rule-action.mdx

```diff
@@ -0,0 +1,152 @@
+- `required_fields` - (Required) List of [FormField](/product/integrations/integration-platform/ui-components/formfield) components the User is required to complete.
+- `optional_fields` - (Optional) List of [FormField](/product/integrations/integration-platform/ui-components/formfield) components the User may complete.
+- `description` - (Optional) Text that will be displayed above the form. Limited to 140 characters.
+
+## Example
+
+```json {filename:schema.json}
+{
+  "elements": [
+    {
+      "type": "alert-rule-action",
+      "title": "Create a Rekall Inc task",
+      "settings": {
+        "type": "alert-rule-settings",
+        "uri": "/sentry/alert-rule",
+        "required_fields": [
+          {
+            "type": "select",
+            "label": "Project",
+            "name": "project",
+            "options": [["1", "cornflakes"]]
+          },
+          {
+            "type": "select",
+            "label": "Assignee",
+            "name": "assignee",
+            "url": "/sentry/alert-rule/options/users/"
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+
+## Issue Alert Request Format
+
+When an issue alert fires, your service will need to read the settings from the alert payload. The `settings` are in `data.issue_alert.settings`. Check out the full [Issue Alert webhook documentation](/product/integrations/integration-platform/webhooks/#issue-alerts) for more information.
+
+### Payload
+
+```json
+{
+  ...
+  "data": {
+    ...
+    "issue_alert": {
+      ...
+      "settings": [
+        {
+          "name": "title",
+          "value": "Ticket Title"
+        },
+        {
+          "name": "description",
+          "value": "Ticket Description"
+        }
+      ]
+    }
+  }
+}
+```
+
+## Metric Alert Request Format
+
+When a metric alert fires, your service will need to read the settings from the alert payload. The `settings` in this example are nested in `data.metric_alert.alert_rule.triggers[1].actions[0].settings`. You will have to identify which trigger and action contains your settings based on the webhook type, so `triggers[1]` and `actions[0]` may not be accurate in your case. Check out the full [Metric Alert webhook documentation](/product/integrations/integration-platform/webhooks/#metric-alerts) for more information.
+
+### Payload
+
+```json
+{
+  “data”: {
+    ...
+    “metric_alert”: {
+      ...
+      “alert_rule”: {
+        ...
+        “triggers”: [
+          ...
+          {
+            ...
+            “actions”: [
+              {
+                ...
+                “settings”: [
+                  {
+                    “name”: “title”,
+                    “value”: “Critical Trigger”
+                  },
+                  {
+                    “name”: “description”,
+                    “value”: “Critical Description”
+                  }
+                ],
+
+              }
+            ]
+          },
+        ],
+
+      },
+    },
+  },
+}
```

Comment:
So there's still some formatting issues here, you can tell since the code block is colourless on the preview, plus the diff on GitHub shows some helpful red highlighting
- The double quotes are probably not actual double-quotes, but 'opening double quotes' and 'closing double quotes"
- JSON hates trailing commas
- The newlines on L132 and L137 can be removed

Reply:
Also a SUPER nit, there should be a `...` above `"data"`